src/UI/.env.local.template

```diff
@@ -9,7 +9,7 @@ AUTH0_ISSUER_BASE_URL='https://dev-326tk4zu.us.auth0.com'
 AUTH0_CLIENT_ID='Al9JO5wSIRNMf8kv5zl2KBW1l5iT2coW'
 # Your Auth0 application's Client Secret
 # This is currently stored in NPD, under Technical Documents - Technical Report - Auth0 Secrets for Vector Atlas
-AUTH0_CLIENT_SECRET='<auth0-client-secret>'
+AUTH0_CLIENT_SECRET='xDAJO3Jh0PKvFeYjDEy01frdktCI5hELOqWWKPTmt_rzu2xWhh8pqvHYtjF2r7Yg'
```

Comment:
Please can you revert this change - secrets should not be checked into the codebase,